go/ql/lib/semmle/go/dataflow/FunctionInputsAndOutputs.qll

```diff
@@ -304,6 +304,13 @@ private class OutParameter extends FunctionOutput, TOutParameter {
     |
       result.(DataFlow::PostUpdateNode).getPreUpdateNode() = arg
     )
+    or
+    exists(DataFlow::Node arg |
+      arg.asExpr() =
+        c.getCall().getImplicitVarargsArgument(index - c.getTarget().getNumParameter() + 1)
+    |
+      result.(DataFlow::PostUpdateNode).getPreUpdateNode() = arg
```

Comment:
Since this is now identical to line 305, push the disjunction inside the exists at line 300